flutter/shell/platform/tizen/channels/app_control.cc

```diff
@@ -369,4 +369,12 @@ AppControlResult AppControl::Reply(AppControl* reply,
                                              result_e);
 }
 
+AppControlResult AppControl::SetAutoRestart(bool enabled) {
+  if (enabled) {
+    return app_control_set_auto_restart(handle_);
```

Comment:
Our target API level is Tizen 5.5. Tizen 7.0-specific API cannot be used in the embedder (especially if it's an internal API).

Can't you just call `app_control_set_auto_restart` using FFI in your app?

Reply:
The robot platform team wants to use this API to set/unset auto restart at runtime.
It's a requirement from Robot Platform team.
I'll discuss it with the developer again.
Thank you for reviewing this PR.